assets/static/js/common.js

```diff
@@ -105,3 +105,7 @@ function showConfirmation(options) //title, message, yesText, noText
         }, options));
     });
 }
+
+function gmapsCallback()
```

Comment:
This seems to not get loaded in time for dataland, or probably any of the vue apps.

Reply:
If we are going to add an empty callback, why not just add `Function.prototype` in those script tags